### PR TITLE
Fix #725 Update global library demo

### LINKED ISSUE #725: Unable to process `unclassified.globalLibraries` configurations
Hi
I'm trying to configure shared libraries but it seems the plugin fails to process them.

Below are the configurations provided. I first configured my shared library on Jenkins and then extracted the following using JCasC plugin. 

```yaml
unclassified:
  globalLibraries:
    libraries:
      - name: "my-shared-lib"
        defaultVersion: "master"
        implicit: true
        retriever:
          modernSCM:
            scm:
              github:
                id: "8704a675-abdf-4574-bda3-ff9bec3c7730"
                repoOwner: "poulad"
                repository: "Jenkins-Pipeline-Tester"
                traits:
                - branchDiscoveryTrait:
                    strategyId: 1
                - originPullRequestDiscoveryTrait:
                    strategyId: 1
                - forkPullRequestDiscoveryTrait:
                    strategyId: 1
```

Error on startup:

```text
io.jenkins.plugins.casc.ConfiguratorException: Invalid configuration elements for type class jenkins.model.GlobalConfigurationCategory$Unclassified : globalLibraries.
Available attributes : administrativeMonitorsConfiguration, artifactManager, casCGlobalConfig, cloud, defaultView, location, masterBuild, myView, nodeProperties, plugin, pollSCM, projectNamingStrategy, quietPeriod, scmRetryCount, shell, usageStatistics, viewsTabBar
	at io.jenkins.plugins.casc.BaseConfigurator.handleUnknown(BaseConfigurator.java:374)
	at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:363)
	at io.jenkins.plugins.casc.BaseConfigurator.check(BaseConfigurator.java:284)
	at io.jenkins.plugins.casc.ConfigurationAsCode.lambda$checkWith$8(ConfigurationAsCode.java:657)
	at io.jenkins.plugins.casc.ConfigurationAsCode.invokeWith(ConfigurationAsCode.java:620)
Caused: io.jenkins.plugins.casc.ConfiguratorException: unclassified: error configuring 'unclassified' with class io.jenkins.plugins.casc.impl.configurators.GlobalConfigurationCategoryConfigurator configurator
	at io.jenkins.plugins.casc.ConfigurationAsCode.invokeWith(ConfigurationAsCode.java:626)
	at io.jenkins.plugins.casc.ConfigurationAsCode.checkWith(ConfigurationAsCode.java:657)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:642)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:545)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configure(ConfigurationAsCode.java:275)
	at io.jenkins.plugins.casc.ConfigurationAsCode.init(ConfigurationAsCode.java:267)
Caused: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at hudson.init.TaskMethodFinder.invoke(TaskMethodFinder.java:104)
Caused: java.lang.Error
	at hudson.init.TaskMethodFinder.invoke(TaskMethodFinder.java:110)
	at hudson.init.TaskMethodFinder$TaskImpl.run(TaskMethodFinder.java:175)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:296)
	at jenkins.model.Jenkins$5.runTask(Jenkins.java:1083)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:214)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused: org.jvnet.hudson.reactor.ReactorException
	at org.jvnet.hudson.reactor.Reactor.execute(Reactor.java:282)
	at jenkins.InitReactorRunner.run(InitReactorRunner.java:48)
	at jenkins.model.Jenkins.executeReactor(Jenkins.java:1117)
	at jenkins.model.Jenkins.<init>(Jenkins.java:921)
	at hudson.model.Hudson.<init>(Hudson.java:85)
	at hudson.model.Hudson.<init>(Hudson.java:81)
	at hudson.WebAppMain$3.run(WebAppMain.java:233)
Caused: hudson.util.HudsonFailedToLoad
	at hudson.WebAppMain$3.run(WebAppMain.java:250)
```

- Jenkins version: 2.150.2
- Plugin version: 1.5
- OS: Docker Container (`FROM jenkins/jenkins:lts`)

### PR DESCRIPTION
**Pull request by poulad**
- [X] **Please describe what you did.**
Updated the global library demo descriptions to include the required support plugin.

- [X] **Link to issue you're working on if there's a relevant one.**
Issue #725 

- [x] **Did you provide a test-case to demonstrate feature actually works / issue is fixed?**
N/A

- [X] **Please also consider adding a line to [CHANGELOG.md](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/CHANGELOG.md)**

### CODE REVIEW

File: demos/workflow-cps-global-lib/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Configure global libraries plugin
 
-Global Pipeline Libraries plugin configuration belongs under `unclassified` root element
+Global Pipeline Libraries plugin configuration belongs under the `unclassified` root element.
+The [`configuration-as-code-support`] plugin needs to be installed as well.
```
Comment (timja) [inline demos/workflow-cps-global-lib/README.md]: @ewelinawilkosz Until the changes have been upstreamed could the plugin just depend on configuration-as-code-support.
For the majority of people I assume they need the support plugin, if you're touching credentials, users, jobs etc.

Not sure if it is actually needed in this case though

**Comment by casz** [comment]
There is no code inside support plugin that references global libraries.
Global libraries ought to work without support plugin.

**Comment by poulad** [comment]
@casz did you try it without the support plugin?

**Comment by casz** [comment]
I ran our integration test after removing the support plugin, works just fine.

**Comment by casz** [comment]
Again @poulad there is no code that references global libraries inside the support plugin

**Pull request closed without merging**